ci/test_utils.py

```diff
@@ -81,58 +81,106 @@ def t_fun_exitCodeAndErrMessage(self, function, exitCode, errMessage, args=[], k
         ''' (Generic) Runs function with given arguments and tests exit code and error message '''
         self.t_fun_exitCodeAndMessage(function, exitCode, BaseTestCaseCapture.MessageType.ERR, errMessage, args, kwargs)
 
-    def t_exitCodeAndLogMessage(self, exitCode, logMessage, args=[], kwargs={}):
+    def t_exitCodeAndLogMessage(self, exitCode, message, args=[], kwargs={}):
         ''' (Generic) Runs predefined function (callfunc) with given arguments and tests exit code and log message '''
-        self.t_fun_exitCodeAndLogMessage(self.callfunc, exitCode, logMessage, args, kwargs)
+        self.t_fun_exitCodeAndLogMessage(self.callfunc, exitCode, message, args, kwargs)
 
-    def t_fun_exitCodeAndLogMessage(self, function, exitCode, logMessage, args=[], kwargs={}):
+    def t_fun_exitCodeAndLogMessage(self, function, exitCode, message, args=[], kwargs={}):
         ''' (Generic) Runs function with given arguments and tests exit code and log message '''
-        self.t_fun_exitCodeAndMessage(function, exitCode, BaseTestCaseCapture.MessageType.LOG, logMessage, args, kwargs)
+        self.t_fun_exitCodeAndMessage(function, exitCode, BaseTestCaseCapture.MessageType.LOG, message, args, kwargs)
 
-    def t_exitCodeAndMessage(self, function, exitCode, messageType, logMessage, args=[], kwargs={}):
+    def t_exitCodeAndMessage(self, function, exitCode, messageType, message, args=[], kwargs={}):
         ''' (Generic) Runs predefined function (callfunc) with given arguments and tests exit code and message of given type '''
-        self.t_fun_exitCodeAndMessage(self.callfunc, exitCode, messageType, logMessage, args, kwargs)
+        self.t_fun_exitCodeAndMessage(self.callfunc, exitCode, messageType, message, args, kwargs)
 
-    def t_fun_exitCodeAndMessage(self, function, exitCode, messageType, logMessage, args=[], kwargs={}):
+    def t_fun_exitCodeAndMessage(self, function, exitCode, messageType, message, args=[], kwargs={}):
         ''' (Generic) Runs function with given arguments and tests exit code and message of given type '''
-        with pytest.raises(SystemExit) as pytest_wrapped_e:
-            function(*args, **kwargs)
-        self.captured = self.capfd.readouterr()
-        self.logs = self.caplog
-        assert pytest_wrapped_e.value.code == exitCode
-
-        if messageType == BaseTestCaseCapture.MessageType.OUT:
-            assert logMessage in self.captured.out
-        elif messageType == BaseTestCaseCapture.MessageType.ERR:
-            assert logMessage in self.captured.err
-        elif messageType == BaseTestCaseCapture.MessageType.LOG:
-            assert logMessage in self.logs.text
-        else:
-            pytest.fail('Uknown MessageType: ' + messageType)
+        self.t_fun_generic(function, SystemExit, str(exitCode), messageType, message, args, kwargs)
 
-    def t_raiseError(self, errorType, errMessage, args=[], kwargs={}):
+    def t_raiseError(self, exceptionType, exceptionValue, args=[], kwargs={}):
```

Comment:
Consider renaming to `t_raiseException` (because you already renamed parameters).